OpenRA.Mods.Cnc/Traits/EnergyWall.cs

```diff
@@ -47,7 +47,7 @@ public void RulesetLoaded(Ruleset rules, ActorInfo ai)
 
 	public class EnergyWall : Building, IObservesVariables, ITick, ITemporaryBlocker
 	{
-		readonly EnergyWallInfo info;
+		public readonly EnergyWallInfo WallInfo;
```

Comment:
In c1cba4ecc18c162be45445bc1f64077c56d804ff we separated `Gate` away from `Building`.  It would be a good idea to do this here too for consistency, and then this can go to `Info` instead of `WallInfo`.

Reply:
Done.